tools/test.py

```diff
@@ -134,7 +134,8 @@ def main():
                 warmup=args.warmup,
                 log_interval=args.log_interval,
                 with_sync=with_sync,
-                file=args.log2file):
+                file=args.log2file,
+                batch_size=model_cfg.data.samples_per_gpu):
```

Comment:
Not all models support batched inference.

Reply:
yes, that model would use default batch-size which is 1.